charts/consul/test/acceptance/framework/config/config.go

```diff
@@ -91,38 +90,48 @@ func (t *TestConfig) HelmValuesFromConfig() (map[string]string, error) {
 	return helmValues, nil
 }
 
-// entImage parses out consul version from Chart.yaml
+type values struct {
+	Global globalValues `yaml:"global"`
+}
+
+type globalValues struct {
+	Image string `yaml:"image"`
+}
+
+// entImage parses out consul version from values.yaml
 // and sets global.image to the consul enterprise image with that version.
 func (t *TestConfig) entImage() (string, error) {
 	if t.helmChartPath == "" {
 		t.helmChartPath = HelmChartPath
 	}
 
-	// Unmarshal Chart.yaml to get appVersion (i.e. Consul version)
-	chart, err := ioutil.ReadFile(filepath.Join(t.helmChartPath, "Chart.yaml"))
+	// Unmarshal values.yaml to current global.image value.
+	valuesContents, err := ioutil.ReadFile(filepath.Join(t.helmChartPath, "values.yaml"))
 	if err != nil {
 		return "", err
 	}
 
-	var chartMap map[string]interface{}
-	err = yaml.Unmarshal(chart, &chartMap)
+	var v values
+	err = yaml.Unmarshal(valuesContents, &v)
 	if err != nil {
 		return "", err
 	}
 
-	appVersion, ok := chartMap["appVersion"].(string)
-	if !ok {
-		return "", errors.New("unable to cast chartMap.appVersion to string")
+	consulImageSplits := strings.Split(v.Global.Image, ":")
+	if len(consulImageSplits) != 2 {
+		return "", fmt.Errorf("could not determine consul version from global.image: %s", v.Global.Image)
 	}
+	consulImageVersion := consulImageSplits[1]
+
 	var preRelease string
 	// Handle versions like 1.9.0-rc1.
-	if strings.Contains(appVersion, "-") {
-		split := strings.Split(appVersion, "-")
-		appVersion = split[0]
+	if strings.Contains(consulImageVersion, "-") {
+		split := strings.Split(consulImageVersion, "-")
+		consulImageVersion = split[0]
 		preRelease = fmt.Sprintf("-%s", split[1])
 	}
 
-	return fmt.Sprintf("hashicorp/consul-enterprise:%s-ent%s", appVersion, preRelease), nil
+	return fmt.Sprintf("hashicorp/consul-enterprise:%s-ent%s", consulImageVersion, preRelease), nil
```

Comment:
Can we add an exception here to images in the `repo/image@sha256:oioi2452345kjhlkh` form? And a test for it as well. 🙏

Reply:
yeah sure! is there a use case for setting these in `values.yaml`? What should be the behavior in that case?

Reply:
In this case, i would expect it to just use the provided image. A specific use case would be me wanting to run acceptance tests for an enterprise feature against a consul image that hasn't been released.

Reply:
I guess it doesnt HAVE to be set in values.yaml but it is much easier to do as compared to updating the CI config, but that's me. Like I mentioned, there is a workaround for that so I wouldn't sweat it.

Reply:
that makes sense! It should be easy to add.

Reply:
youre very kind!! 🙏

Reply:
added!